### PR TITLE
Add to Community Providers section.

### PR DESCRIPTION
**Pull request by cgcardona**
* Add DFK, Swimmer and Dexalot Subnet RPCs
* Add Infura RPCs

Cleaning up https://github.com/ava-labs/avalanche-docs/pull/759

* Formatting changes per Vale suggestions
* Add Mainnet and Testnet sections to ANKR section
* Add EVM/X/C/P API endpoints for both ANKR Mainnet and Testnet APIs

Cleaning up https://github.com/ava-labs/avalanche-docs/pull/1111

* Add Mainnet and Testnet sections to GetBlock section

### CODE REVIEW

File: docs/apis/avalanchego/public-api-server.md [hunk 1]
```diff
@@ -60,5 +85,21 @@ If you have questions, problems or suggestions, come [talk to us](https://chat.a
...
\ No newline at end of file
+Ava Labs offers no warranty/guarantee on the quality or uptime of these community RPC providers
+:::
+
+### Ankr
+
+- For C-Chain API, the URL is `https://rpc.ankr.com/avalanche`.
+- For X-Chain API, the URL is `https://rpc.ankr.com/avalanche-x`.
+- For P-Chain API, the URL is `https://rpc.ankr.com/avalanche-p`.
+- For Fuji Testnet API, the URL is `https://rpc.ankr.com/avalanche_fuji`.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:96]: then what are the endpoints for X/P/C-chain API on Fuji? I think we should list them out.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/public-api-server.md (hunk 1 above)
Comment (cgcardona) [reply docs/apis/avalanchego/public-api-server.md]: Issue resolved

File: docs/apis/avalanchego/public-api-server.md [hunk 2]
```diff
@@ -4,11 +4,18 @@ description: There is a public API server that allows developers to access the A
...
 # Public API Server
 
-There is a public API server that allows developers to access the Avalanche network without having to run a node themselves. The public API server is actually several [AvalancheGo](https://github.com/ava-labs/avalanchego) nodes behind a load balancer to ensure high availability and high request throughput.
+There is a public API server that allows developers to access the Avalanche
+network without having to run a node themselves. The public API server is
+actually several [AvalancheGo](https://github.com/ava-labs/avalanchego) nodes
+behind a load balancer to ensure high availability and high request throughput.
 
-## Using the Public API nodes
+## Using the Public API Nodes
```
Comment (patrick-ogrady) [inline docs/apis/avalanchego/public-api-server.md:12]: You should link out to Subnet APIs too.

Comment (cgcardona) [reply docs/apis/avalanchego/public-api-server.md]: Done https://github.com/ava-labs/avalanche-docs/pull/1098/commits/b1fef276a24ab9b8889801c61be7d871a905eefb

**Review by yulin-dong — DISMISSED** [review]
LGTM

**Pull request merged**